services-geojson/src/main/java/com/mapbox/geojson/utils/GeoJsonUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.geojson.utils;
+
+/**
+ * GeoJson utils class contains method that can be used throughout geojson package.
+ *
+ * @since 4.3.0
+ */
+public class GeoJsonUtils {
+
+  /**
+   * Trims a double value to have only 7 digits after period.
+   *
+   * @param value to be trimed
+   * @return trimmed value
+   */
+  public static double trim(double value) {
+    return Math.round(value * 10000000.0) / 10000000.0;
```

Comment:
Can we do a check for infinity overflow here after multiplication and throw an exception if it occurs?